### ISSUE TITLE
Text box widget

### ISSUE THREAD

**Issue by jackTabsCode** [matter-ecs/plasma#11]
There is currently no text box widget in Matter. I think a first party feature should be provided.

**Comment by Ukendio** [comment]
I agree that we need this, the search feature also needs a textbox, however we need to make sure it can propagate changes to the server.
